drupal_form_api.module

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * @file
+ * Example useage of Drupal Form API module
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function drupal_form_api_menu() {
+  $items = array();
+
+  $items['admin/settings/drupal_form_api'] = array(
+    'title' => 'Drupal Form API Module Settings',
+    'description' => 'This page allows you to administer settings for the module',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('drupal_form_api_admin'),
+    'access arguments' => array('administer drupal_form_api settings'),
```

Comment:
That permission doesn't exist yet, but will work for administrators.

Take a look at https://www.drupal.org/node/553368 and https://api.drupal.org/api/drupal/modules%21system%21system.api.php/function/hook_permission/7 - define `hook_permission` to provide a permission by that name.